specifyweb/specify/views.py

```diff
@@ -545,15 +572,28 @@ def update_record(record: models.Model):
 })
 @login_maybe_required
 @require_POST
-def record_merge(request: http.HttpRequest, model_name: str, old_model_id: int, new_model_id: int) -> http.HttpResponse:
-    """Replaces all the foreign keys referencing the old record ID
-    with the new record ID, and deletes the old record.
+def record_merge(request: http.HttpRequest, model_name: str, new_model_id: int) -> http.HttpResponse:
+    """Replaces all the foreign keys referencing the old record IDs
+    with the new record ID, and deletes the old records.
     """
-    check_permission_targets(None, request.specify_user.id, [ReplaceRecordPT.update, ReplaceRecordPT.delete])
-
-    # TODO: Test reocord merging targeting records other than Agent before allowing generic merging
-    # if model_name.lower() != 'Agent'.lower():
-    #     return http.HttpResponseForbidden(f'Record Merging for the {model_name} table is not yet been fully tested.')
-
-    response = record_merge_fx(model_name, old_model_id, new_model_id)
+    request_params = http.QueryDict(request.META['QUERY_STRING'])
```

Comment:
What is the purpose of this `request_params` object? It's not used anywhere else in the function.